plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/Instrumentable.kt

```diff
@@ -1,21 +1,28 @@
 package io.sentry.android.gradle.instrumentation
 
-import io.sentry.android.gradle.instrumentation.database.AndroidXSQLiteClassVisitor
-import org.objectweb.asm.ClassVisitor
+interface Instrumentable<Visitor> {
```

Comment:
I figured we can't really have it in the old way with enums, cause we'd need to create enums for each type (class, method, field, annotation, etc. visitors), so I switched to an interface here that all those can implement